packages/l/llvm/fetch.lua

```diff
@@ -2,54 +2,52 @@ import("lib.detect.find_tool")
 import("core.project.target")
 
 function main(package, opt)
-    if opt.system then
```

Comment:
why remove it?

Reply:
oops well spoted, this was a test that I mistakenly forgot to revert